src/app/qml/CancelDialog.qml

```diff
@@ -51,7 +51,7 @@ ApplicationWindow {
                     else if (releases.variant.status == Units.DownloadStatus.Writing) 
                         qsTr("Do you want to cancel writing?")
                     else
-                        qsTr("Do you want to cancel validitation of data? This operation is safe.")
+                        qsTr("This operation is safe to be cancelled.")
```

Comment:
You should still ask whether the user wishes to cancel the process. 

→ Do you want to cancel data validation? This operation is safe to be cancelled.